src/rgw/rgw_cache.h

```diff
@@ -190,7 +190,7 @@ struct DataCache {
   } io_type;
 
   struct sigaction action;
-  long long free_data_cache_size;
+  long long free_data_cache_size = 0;
```

Comment:
I suspect we should use int64_t or uint64_t for all these

Reply:
changed to `uint64_t`